### ISSUE TITLE
Error calling create POI.

### ISSUE THREAD

**Issue by Nightwing52** [DartExplore/dallas-by-metro-backend#148]
If we create a POI with no amenities like so,
![image](https://github.com/DartExplore/dallas-by-metro-backend/assets/18405628/2eb02a4b-87d5-4c57-9967-80483a2691c8)
We get the following error.
![image](https://github.com/DartExplore/dallas-by-metro-backend/assets/18405628/327bf8fa-e19e-40a8-ac27-0c6cb31f7cd5)
Which can be traced back to this line of code,
![image](https://github.com/DartExplore/dallas-by-metro-backend/assets/18405628/cf5bce78-1ea9-475a-8f05-7c6f6087b8ee)
Basically, if we pass no amenities we can't convert it into a stream. I propose making a ternary statement to set the list equal to an empty list if no amenities are given.
